src/autocomplete.js

```diff
@@ -502,6 +509,7 @@ yourlabs.Autocomplete.prototype.show = function(html) {
     // Show the inner and outer container only if necessary.
     if (!this.box.is(':visible')) {
         this.box.css('display', 'block');
+        this.fixPosition();
```

Comment:
It's been a few months now so this is hazy in my memory, but I remember there being issues with the box not aligning correctly at times, and adding these two extra calls to `fixPosition()` resolved the issue and ensured that the box is always in the correct position (been running this for months and the alignment has been solid).